ci_deployer/deploy.sh

```diff
@@ -3,13 +3,12 @@
 git clone https://github.com/Open-IoT-Service-Platform/platform-launcher.git -b develop
 cd platform-launcher
 echo y | make update
-helm init --upgrade --wait
 export DOCKERUSER=${DOCKER_USERNAME}
 export DOCKERPASS=${DOCKER_PASSWORD}
 export NODOCKERLOGIN=true
-if [[ $(helm ls -q | grep "${NAME}\$") ]]; then
+if [[ $(helm ls -q --all-namespaces | grep "${NAME}\$") ]]; then
     CMD="upgrade"
-    rm kubernetes/templates/jobs/db_setup.yaml
+    rm kubernetes/templates/jobs/*.yaml
```

Comment:
why are you deleting all jobs here? at least db_upgrade should run for helm upgrade
Also, we can use helm hooks to rule them out e.g. for updates

Reply:
This is included in the current build, I have just included it for completeness, and this is not necessarily part of this PR. We can remove it and have look later. I think the problem was, that jobs cannot be rerun/recreated.
The optimal solution would be to use an operator to manage db updates as well, but I am not sure whether ot would be worth the effort.